### PR TITLE
Rosetta: Add instructions and config to run rosetta-cli tests.

### LINKED ISSUE #9485: Write up instructions on how to test Rosetta in our infrastructure

### PR DESCRIPTION
**Pull request by Sventimir**
`rosetta-cli` is a testing tool for Rosetta implementations released by Coinbase. It allows to automatically verify a running instance with Rosetta specification and validate the contents of the blockchain as presented by the instance. This PR provides instructions on how to manually set up a Mina daemon and Rosetta server, and how to use `rosetta-cli` to test it.

This is documentation-only change.

* Closes #9485

### CODE REVIEW

File: src/app/rosetta/rosetta-cli-config/config.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+   "offline_url":                "http://localhost:3087",
+   "max_offline_connections":    4,
+   "force_retry":                false,
+   "stale_depth":                3,
+   "broadcast_limit":            5,
+   "ignore_broadcast_failures":  false,
+   "clear_broadcasts":           true,
+   "broadcast_behind_tip":       false,
+   "block_broadcast_limit":      5,
+   "rebroadcast_all":            true,
```
Comment (ylecornec) [inline src/app/rosetta/rosetta-cli-config/config.json:31]:
```suggestion
   "rebroadcast_all":            false,
```

File: src/app/rosetta/rosetta-cli-config/config.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ "construction": {
+   "offline_url":                "http://localhost:3087",
+   "max_offline_connections":    4,
+   "force_retry":                false,
+   "stale_depth":                3,
+   "broadcast_limit":            5,
+   "ignore_broadcast_failures":  false,
+   "clear_broadcasts":           true,
+   "broadcast_behind_tip":       false,
+   "block_broadcast_limit":      5,
```
Comment (ylecornec) [inline src/app/rosetta/rosetta-cli-config/config.json:30]:
```suggestion
   "block_broadcast_limit":      0,
```

File: src/app/rosetta/rosetta-cli-config/config.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+ "error_stack_trace_disabled": false,
+ "coin_supported": false,
+ "construction": {
+   "offline_url":                "http://localhost:3087",
+   "max_offline_connections":    4,
+   "force_retry":                false,
+   "stale_depth":                3,
+   "broadcast_limit":            5,
+   "ignore_broadcast_failures":  false,
+   "clear_broadcasts":           true,
```
Comment (ylecornec) [inline src/app/rosetta/rosetta-cli-config/config.json:28]:
```suggestion
   "clear_broadcasts":           false,
```

File: src/app/rosetta/rosetta-cli-config/config.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ "l0_in_memory_enabled": false,
+ "all_in_memory_enabled": false,
+ "error_stack_trace_disabled": false,
+ "coin_supported": false,
+ "construction": {
+   "offline_url":                "http://localhost:3087",
+   "max_offline_connections":    4,
+   "force_retry":                false,
+   "stale_depth":                3,
+   "broadcast_limit":            5,
```
Comment (ylecornec) [inline src/app/rosetta/rosetta-cli-config/config.json:26]:
```suggestion
   "broadcast_limit":            0,
```

**Review by ylecornec — APPROVED** [review]
It looks good to me. I made some suggestions regarding the config for the construction tests.
It does not fully work either but fixes some issues, and is closer to the existing config files from the `src/app/rosetta`directory.

**Review by bkase — APPROVED** [review]

**Comment by Sventimir** [comment]
!ci-build-me

File: src/app/rosetta/README.md [hunk 5]
```diff
@@ -362,7 +362,10 @@ The signer library used by the test agent can be used as a reference for further
 
 ### Rosetta CLI Validation
 
-The Data API is fully validated using the official `rosetta-cli` against private networks that issue every different type of transaction. There are no reconcilliation errors. We are in the middle of verifying reconcilliation errors against devnet.
+The Data API is fully validated using the official `rosetta-cli`
+against private networks that issue every different type of
+transaction. For instructions on how to run these tests manually,
+see `REAMDE-rosetta-cli.md`.
```
Comment (joaosreis) [inline src/app/rosetta/README.md:368]: Small typo here on the word `README`. Also, should this be linked to the respective file?

Comment (Sventimir) [reply src/app/rosetta/README.md]: Thanks, corrected.

**Comment by Sventimir** [comment]
!ci-build-me

**Pull request merged**